Readme.md

```diff
@@ -24,22 +24,28 @@ Here's a trivial example:
 
 ```rust
 extern crate assert_cli;
-assert_cli::assert_cli_output("echo", &["42"], "42").unwrap();
+fn main() {
+  assert_cli::assert_cli_output("echo", &["42"], "42").unwrap();
+}
 ```
 
 Or if you'd rather use the macro:
 
 ```rust,ignore
 #[macro_use] extern crate assert_cli;
-assert_cli!("echo", &["42"] => Success, "42").unwrap();
-assert_cli!("black-box", &["--special"] => Error 42, "error no 42\n").unwrap()
+fn main() {
+  assert_cli!("echo", &["42"] => Success, "42").unwrap();
+  assert_cli!("black-box", &["--special"] => Error 42, "error no 42\n").unwrap()
```

Comment:
If you find use any other, universally available cli tool (` exit`?) you can get rid of `ignore` :)